### ISSUE TITLE
sdkShareDelegate is weak defined and it is always nil

### ISSUE THREAD

**Issue by bybash** [facebook/facebook-sdk-swift#256]
Hi,

I'm using FacebookShare 0.3.2 in my project and when I want to use completion callback, I realized that it is not called. I checked the code and sdkShareDelegate variable is defined as weak. After debugging I saw that after initiation, its value was always nil. So callbacks were not called. I checked pre-commits and saw that your previous codes don't have "weak" identifier for sdkShareDelegate. After deleting weak identifier from code, callbacks started to work. 

At ShowDialog.swift code was
```swift
public final class ShareDialog<Content: ContentProtocol>: ContentSharingProtocol, ContentSharingDialogProtocol {
  private let sdkSharer: FBSDKShareDialog
  private weak var sdkShareDelegate: SDKSharingDelegateBridge<Content>?
```

I changed it to: 
```swift
public final class ShareDialog<Content: ContentProtocol>: ContentSharingProtocol, ContentSharingDialogProtocol {
  private let sdkSharer: FBSDKShareDialog
  private var sdkShareDelegate: SDKSharingDelegateBridge<Content>?
```

I don't know, is this a bug or project dependent problem but it created problems in my projects and I just wanted to write it. 

Thank you.
Bybash

**Comment by codytwinton** [comment]
@bybash - Thanks for bringing up this issue! I'll do my best to push a change to resolve this soon.

In the meantime, feel free to submit a PR! I'll approve asap.

**Comment by bybash** [comment]
Thanks, @codytwinton 
I submitted a PR, you can check it.
Regards.

**Comment by codytwinton** [comment]
@bybash - Thanks for submitting #258 PR. As soon as you sign the CLA, I can approve it!

**Comment by bybash** [comment]
Sorry @codytwinton 
I missed it, right now I signed the CLA.

**Comment by codytwinton** [comment]
Fixed with #258 PR. Thanks @bybash for the contribution!

**Comment by bybash** [comment]
👍🏻🙏
